setup.py

```diff
@@ -23,7 +23,7 @@ def package_data(pkg, root_list):
 
 setup(
     name='xblock-image-explorer',
-    version='0.5.0',
+    version='0.6.0',
```

Comment:
Sorry!  I should noticed that we already have a [tag called `v0.6.0`](https://github.com/edx-solutions/xblock-image-explorer/tree/v0.6.0) (but forgot to bump this version at that time).  Could you make this `0.7.0` instead?